server-code/WorkPackages.js

```diff
@@ -54,6 +54,14 @@ function getWorkPackageObj(wbsNum) {
         end: rowData[findIdx("End", headers)],
         spend: rowData[findIdx("Spend", headers)],
     };
+
+    //now validate fields in the work package
+    //add more as required 
+    if (!workPackage.lead) {workPackage.lead = "—";}
+    if (!workPackage.dependencies) {workPackage.dependencies = "—";}
+    if (!workPackage.deliverable) {workPackage.deliverable = "—";}
+    if (!workPackage.changes) {workPackage.changes = "—";}
```

Comment:
Right now, since you add a '—' when the changes field is empty, when `buildOrderedListHTML()` is called in `getWorkPackageHtml()`, `text` will not be null and so won't return '—'.
```suggestion
```

Reply:
Oh yes I missed this! It will instead return `<ol>-</ol>` which is not what is needed. I have committed the change.